application/MainWindow.cpp

```diff
@@ -156,8 +156,13 @@ MainWindow::MessageReceived(BMessage* message)
 
         case MSG_HELP:
         {
-        	char const* args[] = {"http://haikuarchives.github.io/DocumentViewer/help.html", 0};
-        	be_roster->Launch("text/html", 1, args);
+        	BPathFinder pathFinder;
+			BPath path;
```

Comment:
What happened to the indentation here? These lines should start on the same column.